Mage/src/main/java/mage/abilities/keyword/DisturbAbility.java

```diff
@@ -48,6 +52,16 @@ public DisturbAbility(Card card, String manaCost) {
         this.getManaCostsToPay().clear();
         this.addManaCost(new ManaCostsImpl(manaCost));
         this.addSubAbility(new TransformAbility());
+
+        SpellAbility ability = secondSideCard.getSpellAbility();
+        for (Target target : ability.getTargets()) {
+            this.addTarget(target);
```

Comment:
One thing I'm not sure about is whether it's necessary to defensively copy the Target and Effect objects that we're "borrowing" from the back side (i.e. this.addTarget(target.copy())). It seems to work without the defensive copies (I cast a Disturb card in aura mode, bounced it to my hand, got the card into my graveyard, and cast the same card again in aura mode, and everything seemed to go fine) but I don't know if it's actually correct or "safe".